servo-core/src/main/java/com/netflix/servo/stats/StatsBuffer.java

```diff
@@ -122,7 +122,11 @@ public void computeStats() {
       sumSquares += values[i] * values[i];
     }
     mean = (double) total / curSize;
-    variance = (sumSquares / curSize) - (mean * mean);
+    if (curSize == 1) {
```

Comment:
It would be good to add a test case for current size of 1. At first glance none of the existing tests will check this branch.